src/main/java/com/yugabyte/sample/common/CmdLineOpts.java

```diff
@@ -43,29 +46,29 @@
   public static UUID loadTesterUUID;
 
   // The various apps present in this sample.
-  public static enum AppName {
-    CassandraHelloWorld,
-    CassandraKeyValue,
-    CassandraRangeKeyValue,
-    CassandraBatchKeyValue,
-    CassandraBatchTimeseries,
-    CassandraTransactionalKeyValue,
-    CassandraTransactionalRestartRead,
-    CassandraStockTicker,
-    CassandraTimeseries,
-    CassandraUserId,
-    CassandraPersonalization,
-    CassandraSecondaryIndex,
-    CassandraUniqueSecondaryIndex,
-    RedisKeyValue,
-    RedisPipelinedKeyValue,
-    RedisHashPipelined,
-    RedisYBClientKeyValue,
-    SqlInserts,
-    SqlUpdates,
-    SqlSecondaryIndex,
-    SqlSnapshotTxns,
-  }
+  private final static List<String> HELP_WORKLOADS = ImmutableList.of(
```

Comment:
We could make this a list of classes and then call `.getSimpleName()` in a loop to avoid repeating it in code.